lib/routemaster/event_index.rb

```diff
@@ -0,0 +1,18 @@
+module Routemaster
+  class EventIndex
+    attr_reader :url, :store
+
+    def initialize(url, store: Config.cache_redis)
+      @url = url
+      @store = store
+    end
+
+    def increment
+      store.incr("event_index:#{url}").to_i
+    end
+
+    def current
+      (store.get("event_index:#{url}") || 0).to_i
```

Comment:
Would be nice to extract out the key into a function rather than repeating it.

Also we _may_ want to consider putting this into a sharded hash as we'll have hundreds of thousands of keys (the logistics dashboard stores around 700k of them) and that would save a bunch of memory.